lib/skyux-config.js

```diff
@@ -1,5 +1,31 @@
 const jsonUtils = require('./json-utils');
 
+async function updatePlugins(config) {
+  const packageJson = await jsonUtils.readJson('package.json');
+
+  packageJson.devDependencies = packageJson.devDependencies || {};
+
+  const installedPlugins = Object.keys(packageJson.devDependencies).map((packageName) => {
```

Comment:
Is the `.map` necessary if it's just returning the same property you're giving it?

Reply:
Nope! Silly mistake. Removed.